main.js

```diff
@@ -31,14 +31,23 @@ function setText() {
   wordList = [];
   currentWord = 0;
   correctKeys = 0;
-  inputField.value = '';
   timerActive = false;
   clearTimeout(timer);
   textDisplay.style.display = 'block';
   inputField.className = '';
 
   switch (typingMode) {
+		case 'custom':

```

Comment:
Some loose indentation here. You are using hard tabs with a length of 4 while the source document is using soft tabs (spaces) with a length of 2. I would recommend changing the tabs in this file to 2 spaces.

Reply:
Noted. Converted all tabs to spaces.